README.md

```diff
@@ -71,6 +71,10 @@ saveBinaryFile = case getPir $$(PlutusTx.compile [|| mkValidator ||]) of
                    Nothing  -> error "plutus compilation failed"
 ```
 
+Note that you _do not_ need to import Pirouette to get a `.pir` or a `.flat` file from your contract,
+The `getPir` function comes from `plutus-core`.
+
```

Comment:
Should we add an example `savePirFile` function?

Reply:
Why not?! I'll add it